amplify.yml

```diff
@@ -125,6 +125,25 @@ applications:
         baseDirectory: .nuxt
         files:
           - "**/*"
+      cache:
+        paths:
+          - node_modules/**/*
+
+  - appRoot: solidjs-tailwind
+    frontend:
+      phases:
+        preBuild:
+          commands:
+            - nvm install --lts=gallium
+            - pnpm install
```

Comment:
Not sure if nvm supplies `pnpm` by default. Do we need to install it?

Reply:
I think we need to install it, no first class support yet

Reply:
@hdJerry what's the update on this?

Reply:
We can use nvm then.